packages/roosterjs-cross-window/package.json

```diff
@@ -6,5 +6,5 @@
         "roosterjs-editor-dom": ""
     },
     "main": "./lib/index.ts",
-    "version": "1.2.0"
+    "version": "1.4.0"
```

Comment:
I published a 1.3.0 before but I realized it doesn't work since new version for main packages are not published yet. So I deprecated it. Now we need to use 1.4.0 instead